smos-docs-site/pages/concepts.markdown

```diff
@@ -0,0 +1,70 @@
+---
+title: SMOS Concepts
+date: 2020-04-08
+---
+
+GTD
+: GTD is a time management method by David Allen (see [Getting This
+  Done](https://en.wikipedia.org/wiki/Getting_Things_Done) on
+  Wikipedia).
+  SMOS is supposed to be a tool to implement GTD.
```

Comment:
I'd rather not mention GTD like this.
While smos takes a lot of inspiration from GTD, but I'm a bit scared about trademark issues.
I call it "self-management" and refer to GTD as "I drew inspiration from it".
It's also true that smos adds some stuff that's not in GTD, like the automated reporting.